src/Tmds.Ssh/SftpClient.File.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Buffers;
+using System.Buffers.Binary;
+
+namespace Tmds.Ssh
+{
+    public class SftpFile
+    {
+        private byte[] handle;
+        internal SftpFile(byte[] handle)
+        {
+            this.handle = handle;
+        }
+    }
+
+    public enum SftpOpenFlags
+    {
+        Read = 0x00000001,
+        Write = 0x00000002,
+        Append = 0x00000004,
+        CreateNewOrOpen = 0x00000008,
+        Truncate = 0x00000010,
+        CreateNew = 0x00000020 | CreateNewOrOpen,
+    }
+
+    public partial class SftpClient
+    {
+        public async Task<SftpFile> OpenFileAsync(string path, SftpOpenFlags openFlags)
```

Comment:
Let's change the return type to `ValueTask<SftpFile>`.
I think you can just update the return type, and it may work.